tests/test_blend.py

```diff
@@ -79,26 +72,21 @@ def test_init(self):
         assert blend.frame.psfs == frame.psfs
 
         # Test init with psfs
-        frame = scarlet.Frame(images.shape, psfs=target_psf[None], channels=channels)
-        obs1 = scarlet.Observation(images1, psfs=psfs1, channels=channels1).match(frame)
-        obs2 = scarlet.Observation(images2, psfs=psfs2, channels=channels2).match(frame)
-        observations = [obs1, obs2]
-        sources = [scarlet.PointSource(coord, frame, observations) for coord in coords]
+        frame = scarlet.Frame(images.shape, psfs=target_psf[None])
+        obs = scarlet.Observation(images, psfs=psfs).match(frame)
+        sources = [scarlet.PointSource(frame, coord, observation) for coord in coords]
 
         with pytest.raises(AttributeError):
-            obs1.psfs_fft
-        with pytest.raises(AttributeError):
-            obs2.psfs_fft
-        blend = scarlet.Blend(frame, sources, observations)
+            obs.psfs_fft
```

Comment:
You can remove this test (and the pytest import at the top of the file). This test was in place when we ere using `Blend` to do the PSF matching and checked to make sure that the PSFs had't been matched yet. We don't use `psfs_fft` anymore, so this is a useless test now.